serialize.go

```diff
@@ -514,6 +514,23 @@ func DeserializeQueryAndArray(context *Context, buffer *Buffer, serializationTyp
 	return array, query, nil
 }
 
+// HandleLoadArraySchemaRequest Pass the array and serialized request to core which returns the serialized response.
+func HandleLoadArraySchemaRequest(array *Array, serializationType SerializationType, request *Buffer) (*Buffer, error) {
```

Comment:
Looking at a few of our functions above, most of them seem to put the request in front of the serialization type in the parameter list.

Reply:
Done